flake.nix

```diff
@@ -7,6 +7,7 @@
     utils.url = "github:kreisys/flake-utils";
     bitte-cli.url = "github:input-output-hk/bitte-cli";
     hydra.url = "github:NixOS/hydra";
+    hydra.inputs.nixpkgs.url = "nixpkgs/nixos-21.05-small";
```

Comment:
why not `hydra.inputs.nixpkgs.follows = "nixpkgs"`?

Reply:
because this is the value that nix has, it just seems that it somehow cannot resolve the global registry entry on my machine since i set it to a git checkout and the behavior on nested flake inputs may be buggy.
I'm really not sure what the issues is myself, but i couldn't run `nix flake lock --update-input bitte` without this in `infra-ops`.